addons/survey/models/survey_question.py

```diff
@@ -633,8 +680,7 @@ def _get_stats_summary_data_choice(self, user_input_lines):
             'partial_inputs_count': len(partial_inputs),
         }
 
-    def _get_stats_summary_data_numerical(self, user_input_lines):
-        all_values = user_input_lines.filtered(lambda line: not line.skipped).mapped('value_numerical_box')
+    def _get_stats_summary_data_numerical(self, all_values):
```

Comment:
You may receive the field_name to check for mapped and keep the "skipped" in the method, like
```
    def _get_stats_summary_data_numerical(self, user_input_lines, fname='value_numerical_box'):
        all_values = user_input_lines.filtered(lambda line: not line.skipped).mapped(fname)
```